PowerFGT/Public/cmdb/webfilter/urlfilter.ps1

```diff
@@ -0,0 +1,441 @@
+﻿#
+# Copyright 2022, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+# Copyright 2022, Cédric Moreau <moreaucedric0 dot gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+function Add-FGTWebfilterUrlfilter {
+
+    <#
+        .SYNOPSIS
+        Add a FortiGate URL Filter
+
+        .DESCRIPTION
+        Add a FortiGate URL Filter
+
+        .EXAMPLE
+        Add-FGTWebfilterUrlfilter -name myURL1 -id 1 -name MyURL -comment "Added by PowerFGT"
+
+        Add URL Filter object named MyURL with comment
+
+        .EXAMPLE
+        Add-FGTWebfilterUrlfilter -name myURL1 -id 1 -name MyURL -comment "Added by PowerFGT" -url_id 1 -url_type simple -url powerfgt.com -action allow -status enable
+
+        Add URL Filter object named MyURL with an url (URL : powerfgt.com, type : simple, action : allow, status : enable)
+
+        .EXAMPLE
+        Add-FGTWebfilterUrlfilter -name myURL1 -id 1 -name MyURL -comment "Added by PowerFGT" -url_id 1 -url_type wildcard -url *powerfgt.com -action block -status enable
+
+        Add URL Filter object named MyURL with an url (URL : *powerfgt.com, type : wildcard, action : block, status : enable)
+
+    #>
+
+    Param(
+        [Parameter (Mandatory = $false)]
+        [string]$id,
```

Comment:
Mandatory ?

Reply:
It takes the next ID available, so it doesn't need to be mandatory or specified